PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -9,10 +9,7 @@
 """
 from __future__ import absolute_import, print_function
 
-import pkg_resources
-
 import ast
-import dis
```

Comment:
Are you used to `gui gui` and `git citool` already? These allow to select even single lines for inclusion or exclusion from the patch. I really like this mighty tools.